e2e/fixtures/fdb_cluster.go

```diff
@@ -394,6 +394,8 @@ func (fdbCluster *FdbCluster) UpdateClusterSpecWithSpec(desiredSpec *fdbv1beta2.
 
 		specUpdated := equality.Semantic.DeepEqual(fetchedCluster.Spec, *desiredSpec)
 		log.Println("UpdateClusterSpec: specUpdated:", specUpdated)
+		fmt.Printf("FetchedSpec %+v\n", fetchedCluster.Spec)
```

Comment:
Please remove this or change it to `log.Println` we don't want to use fmt here.

Reply:
I prefer to change it to log.Printf() because  log.Println() doesn't include the struct's field name. Because Spec is a really large struct, it is super hard to debug w/o the field name. 

Using "%+v" makes sure the output has the struct's field name. 

I'm open to alternatives that can output the struct's field name.